lib/Doctrine/ODM/PHPCR/Translation/TranslationStrategy/AttributeTranslationStrategy.php

```diff
@@ -176,7 +176,7 @@ public function getLocalesFor($document, NodeInterface $node, ClassMetadata $met
         $locales = array();
         foreach ($node->getProperties("*{$this->prefix}*") as $prop) {
             $matches = null;
-            if (preg_match('/' . $this->prefix . ':(..)-[^-]*/', $prop->getName(), $matches)) {
+            if (preg_match('/' . $this->prefix . ':([a-z]{2}_?[a-z]{0,2})-[^-]*/', $prop->getName(), $matches)) {
```

Comment:
But yeah, it is untested, so whatever it should only match `xx_xx` and `xx` finally.